src/main/java/decodex/ui/messages/ErrorMessages.java

```diff
@@ -22,7 +22,8 @@
     public static final String TOO_MANY_COMMAND_ARGUMENTS = "Too many command arguments";
     public static final String INVALID_LIST_CATEGORY = "Invalid list category";
     public static final String UNKNOWN_MODULE_NAME = "Unknown module name";
-    public static final String INVALID_MODULE_PARAMETERS = "Invalid module parameters";
+    public static final String INVALID_ROTENCODE_PARAMETER = "The rotation offset must be between -2147483648 (MIN_INT)"
+            + " and 2147483647 (MAX_INT)";

```

Comment:
Perhaps maybe include `must be an integer between...`? This is so that they do not mistakenly put floating point numbers.